Makefile

```diff
@@ -6,6 +6,9 @@ SHELL = /usr/bin/env bash -o pipefail
 MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 
+# Container Engine to be used for building image and with kind
+CONTAINER_ENGINE ?= docker
```

Comment:
nitpick: `CONTAINER_ENGINE = docker` is enough as the variable has not been defined in the same makefile. Note that it still allows overriding it with `make target CONTAINER_ENGINE=podman`

Reply:
I am going to say this does not work in manner that is useful. If you use `make target CONTAINER_ENGINE=podman` works but, at least in fish, setting the `CONTAINER_ENGINE` as a environment variable and then calling `make target` does not pass the correct value. While the `?=` does allow this to work.